tensorboard/webapp/metrics/views/right_pane/settings_view_container.ts

```diff
@@ -66,6 +67,10 @@ import {HistogramMode, TooltipSort, XAxisType} from '../../types';
 export class SettingsViewContainer {
   constructor(private readonly store: Store<State>) {}
 
+  readonly isImageSupportEnabled$ = this.store.select(
```

Comment:
After writing that long comment about how you are correct to use getFeatureFlagsIsLoaded in tandem with feature flag selectors, I only now see you've done the opposite here.

Should we also use getFeatureFlagsIsLoaded here? Or is there a reason not to do it here that doesn't apply to MetricsDataSource?